.storybook/components/DesignTokens/TokensTable.stories.ts

```diff
@@ -0,0 +1,13 @@
+import type { StoryObj } from '@storybook/react';
+import { TokensTable as TokensTableComponent } from './TokensTable';
+
+export default {
+  title: 'Design Tokens/Tokens Table',
+  component: TokensTableComponent,
+  axe: {
+    // For documentation purposes only
+    skip: true,
+  },
+};
```

Comment:
Should be nested under `parameters`?

```suggestion
  parameters: {
    axe: {
      // For documentation purposes only
      skip: true,
    },
  },
};
```

Also wonder if we should skip Chromatic, as well.

Reply:
yah, having chromatic for these will not be as useful since this particular thing is more of an interactive utility. Any actually useful changes detected here will also show up in other ones we capture currently.